src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/table/table.js

```diff
@@ -235,17 +276,46 @@ class TableUI extends PureComponent {
                 defaultMessage="Delete"
               />
             </EuiButton>,
-            <EuiButton
-              key="exportSO"
-              iconType="exportAction"
-              onClick={onExport}
-              isDisabled={selectedSavedObjects.length === 0}
+            <EuiPopover
+              key="exportSOOptions"
+              button={button}
+              isOpen={this.state.isExportPopoverOpen}
+              closePopover={this.closeExportPopover}
             >
-              <FormattedMessage
-                id="kbn.management.objects.objectsTable.table.exportButtonLabel"
-                defaultMessage="Export"
-              />
-            </EuiButton>,
+              <EuiFormRow
+                label={(
+                  <FormattedMessage
+                    id="kbn.management.objects.objectsTable.exportObjectsConfirmModal.exportOptionsLabel"
+                    defaultMessage="Options"
+                  />
+                )}
+              >
+                <EuiSwitch
+                  name="includeReferencesDeep"
+                  label={(
+                    <FormattedMessage
+                      id="kbn.management.objects.objectsTable.exportObjectsConfirmModal.includeReferencesDeepLabel"
+                      defaultMessage="Include dependencies"
```

Comment:
I wonder if "dependencies" makes sense. Do we use this terminology elsewhere? What do you think about "related objects"?

Reply:
I spoke with @gchaps above and came up with `Include dependencies`, I'm open to changing the terminology.

Reply:
I chose "dependencies" because it is common terminology.  However, if "Include related objects" is more specific, then I'm all for it.

Reply:
I have made the changes, attached is the updated UI.

<img width="505" alt="Screen Shot 2019-04-08 at 9 07 19 PM" src="https://user-images.githubusercontent.com/3694571/55766558-27e2e280-5a43-11e9-8080-ddcc68904c20.png">
<img width="325" alt="Screen Shot 2019-04-08 at 9 07 30 PM" src="https://user-images.githubusercontent.com/3694571/55766559-27e2e280-5a43-11e9-99c6-ae7e34d910f7.png">